Cargo.toml

```diff
@@ -15,6 +15,12 @@ name = "rusoto"
 readme = "README.md"
 repository = "https://github.com/rusoto/rusoto"
 version = "0.24.0"
+homepage = "https://www.rusoto.org/"
+categories = [ "development-tools" ]
```

Comment:
I watched the [categories PR](https://github.com/rust-lang/crates.io/pull/488) and didn't see any category Rusoto really fits in.  It's not necessarily a development tool because it can (and should!) be used for running Rust projects interacting with AWS.  There was a `cloud` category but i[t was removed.](https://github.com/rust-lang/crates.io/pull/488#issuecomment-265649715)

It's also not API bindings because Rusoto is a native Rust project.  If there was an SDK category that'd be great, but it's not there and I haven't asked for one of those in the crates.io repo.

Anyone have opinions on if this category helps or hurts?  I'm ambivalent on it.